content/5_Plat/Binary_Jump.mdx

```diff
@@ -129,6 +129,42 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+import math
+
+
+n, q = map(int, input().split())
+
+
+def kth_ancestor(x: int, k: int) -> int:  # return the kth ancestor of x
```

Comment:
```suggestion
def kth_ancestor(x: int, k: int) -> int:
	""":return the kth ancestor of x"""
```

Reply:
actually now that i think about it it's better it you took the code out of the function and just put it in main

Reply:
Done.